src/src/pages/capabilities/capabilityTags/capabilityTagsSubForm.js

```diff
@@ -19,6 +19,21 @@ import JsonSchemaContext from "../../../JsonSchemaContext";
  * Custom Widgets and Fields
  */
 
+function CustomFieldTemplate(props) {
+  const { id, label, required, rawDescription, children } = props;
+  return (
+    <div className={styles.field}>
+      {required ? <span className={styles.bold}>*</span> : null}
+      <label htmlFor={id}>{label}</label>
+      <br />
+      {rawDescription}
+      {children}
+      {/*errors*/}
+      {/*help*/}
```

Comment:
for debugging (then please add a comment)? Or should this be removed?

Reply:
Actually, I wanted this for future us'es, to indicate that we have more fields to play with should we need it -- in case people are lazy and don't actually read the documentation.

I am already considering using 'help', for allowing field specific documentation links or guidelines. However, this requires schema changes, and I don't know how happy we are about this.

Reply:
ok good to know. Can you maybe just add a comment or even link to documentation.

Reply:
removed these particular lines and just added a comment